packages/framework/src/Component/Cdn/CdnFacade.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Cdn;
+
+use Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+
+class CdnFacade
+{
+    private ?string $cdnDomain = null;
+
+    public function __construct(
+        ?string $cdnDomain,
+        protected readonly Domain $domain,
+    ) {
+        // When you do not want to use CDN, it is used value '//' as workaround by https://github.com/symfony/symfony/issues/28391
```

Comment:
My tendency to not have comments has blast here as this comment is at least in three places :D Would not be easier to have this in one place? For example - here in constructor `if $cdnDomain === null ? '//' ...`